fhir-server/src/main/java/au/csiro/pathling/aggregate/FreshAggregateExecutor.java

```diff
@@ -78,7 +77,7 @@ public FreshAggregateExecutor(@Nonnull final Configuration configuration,
   @Nonnull
   @Override
   public AggregateResponse execute(@Nonnull final AggregateRequest query) {
-    log.info("Executing request");
+    log.info("Executing request: " + query);
```

Comment:
More efficient to to use: 
```
 log.info("Executing request: {}",  query);
 ```

Reply:
See a47ccd7354c37bdf0f9381b8ddd727d22343dd60.